packages/components/src/internal/components/chart/BarChartViewer.tsx

```diff
@@ -140,7 +141,7 @@ export class BarChartViewer extends PureComponent<Props, State> {
             let url = getAppURL(row, evt);
 
             if (chart.filter !== undefined) {
-                const dt = moment().add(chart.filter, 'days').format(getDateFormat().toUpperCase());
+                const dt = formatDate(addDays(new Date(), 1), getDateFNSDateFormat());
```

Comment:
this looks to be hardcoded to add 1 day. Should this be using the `chart.filter` value from CHART_SELECTORS?

Reply:
I'm going to leave this as-is as I'm just looking to do a direct logical translation here.

Reply:
I'm not sure what you mean. If I'm reading the previous line correct, wasn't `moment().add(chart.filter, 'days')` adding the `chart.filter` number of days?

Reply:
Derp. I completely read this wrong. I will get this fixed.

Reply:
Fixed.